sdcm/utils/docker_utils.py

```diff
@@ -78,6 +78,12 @@ def __bool__(self):
         return self.full is not None
 
 
+class INodeWithContainerManager(Protocol):
```

Comment:
Why this needed as Protocol?

Reply:
Protocol is here for static type checking, it's a class that describes an interface